pennylane/tape/tape.py

```diff
@@ -184,7 +184,8 @@ def stop_at(obj):  # pylint: disable=unused-argument
         (tape._ops + diagonalizing_gates, new_ops),
         (diagonal_measurements, new_measurements),
     ]:
-        for obj in queue:
+        for wrapped_obj in queue:
+            obj = wrapped_obj.obj
```

Comment:
At this point, shouldn't we be dealing with the objects themselves?

Reply:
Thanks for pointing that out :)